Framework/Geometry/test/MeshObjectTest.h

```diff
@@ -235,13 +235,6 @@ class MeshObjectTest : public CxxTest::TestSuite {
     TS_ASSERT(cloned);
   }
 
-  void testTooManyVertices() {
```

Comment:
I think an arbitrary limit need to be put in place for this one, as just limiting it to uint32_t causes my test to bad alloc when I try to make a vector large enough to test it with, Ill think of one to set now.

Reply:
Ah ok. Yes, the bad_alloc is a good point. Infact even if V3D was just 48 bytes, most systems would struggle to provide enough memory, though some systems could!

I would not artificially limit the size. Either, refactor the code so that you can test just your error function - pass a `size_t` as input. Or delete the test as you current have done.

Reply:
Yes. If you have enough memory you will not get a `bad_alloc`, but you would still get out of bounds issues.

Reply:
ok, so I'll just pass test passing a size_t to the check function thats larger than 2^32.

Reply:
👍 That would also need to be exposed in a way it could be tested.

Reply:
ist `checkVertexLimit` only in a namespace not a class? shouldnt it already be available to so long as `MeshObjectCommon.h` is `#included`?

Reply:
As long as it's testable by itself.